app/views/appointments/index.csv.erb

```diff
@@ -1,8 +1,9 @@
-<%= [
+<% csv = CSV.generate do |csv| %>
+<% csv << [
```

Comment:
The ERB tags in this block don't output anything, so can we indent them for readability?

```
<% csv = CSV.generate do |csv| %>
  <% csv << [... %>
  <% end %>
<% end.html_safe %>
```